ui/app/src/components/PreviewComponentsPanel/PreviewComponentsPanel.tsx

```diff
@@ -123,10 +117,12 @@ export const ComponentsPanelUI: React.FC<ComponentsPanelUIProps> = (props) => {
     dispatch(
       batchActions([
         setContentTypeFilter(menuContext.contentType.id),
-        pushToolsPanelPage(
-          createToolsPanelPage({ id: 'previewBrowseComponentsPanel.title' }, [
-            createWidgetDescriptor({ id: 'craftercms.components.PreviewBrowseComponentsPanel' })
-          ])
+        pushIcePanelPage(
+          createToolsPanelPage(
+            { id: 'previewBrowseComponentsPanel.title' },
+            [createWidgetDescriptor({ id: 'craftercms.components.PreviewBrowseComponentsPanel' })],
+            'pageBuilderPanel'
```

Comment:
You sure you pulled? This has a new ID, it's icePanel. Check types after pulling merging. This shouldn't even compile.

Reply:
Same for the other instances below (or on other files).